pyproject.toml

```diff
@@ -1,83 +1,7 @@
 [tool.black]
 line-length = 90
 target-version = ["py36"]
-force-exclude = '''
-(
-  # default black exclude
-  /(\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|\.svn|_build|buck-out|build|dist)/
-
-  # moabb specific files
-  | setup.py
-  | moabb/run.py
-  | moabb/__init__.py
-  | moabb/utils.py
-  | moabb/analysis/results.py
-  | moabb/analysis/plotting.py
-  | moabb/analysis/__init__.py
-  | moabb/analysis/meta_analysis.py
-  | moabb/evaluations/__init__.py
-  | moabb/evaluations/evaluations.py
-  | moabb/evaluations/base.py
-  | moabb/datasets/ssvep_mamem.py
-  | moabb/datasets/ssvep_nakanishi.py
-  | moabb/datasets/fake.py
-  | moabb/datasets/download.py
-  | moabb/datasets/__init__.py
-  | moabb/datasets/alex_mi.py
-  | moabb/datasets/physionet_mi.py
-  | moabb/datasets/mpi_mi.py
-  | moabb/datasets/ssvep_exo.py
-  | moabb/datasets/gigadb.py
-  | moabb/datasets/utils.py
-  | moabb/datasets/bnci.py
-  | moabb/datasets/braininvaders.py
-  | moabb/datasets/schirrmeister2017.py
-  | moabb/datasets/epfl.py
-  | moabb/datasets/upper_limb.py
-  | moabb/datasets/Zhou2016.py
-  | moabb/datasets/Weibo2014.py
-  | moabb/datasets/ssvep_wang.py
-  | moabb/datasets/base.py
-  | moabb/datasets/bbci_eeg_fnirs.py
-  | moabb/tests/analysis.py
-  | moabb/tests/download.py
-  | moabb/tests/datasets.py
-  | moabb/tests/__init__.py
-  | moabb/tests/paradigms.py
-  | moabb/tests/evaluations.py
-  | moabb/tests/util_tests.py
-  | moabb/pipelines/classification.py
-  | moabb/pipelines/__init__.py
-  | moabb/pipelines/csp.py
-  | moabb/pipelines/features.py
-  | moabb/pipelines/utils.py
-  | moabb/paradigms/p300.py
-  | moabb/paradigms/__init__.py
-  | moabb/paradigms/motor_imagery.py
-  | moabb/paradigms/ssvep.py
-  | moabb/paradigms/base.py
-  | pipelines/FBCSP.py
-  | pipelines/TSSVM.py
-  | pipelines/CSP_svm_search.py
-  | pipelines/LogVar.py
-  | pipelines/WTRCSP.py
-  | docs/source/conf.py
-  | examples/plot_cross_subject_ssvep.py
-  | examples/plot_within_session_p300.py
-  | examples/plot_cross_session_ssvep.py
-  | examples/plot_cross_session_motor_imagery.py
-  | examples/plot_cross_session_multiple_datasets.py
-  | examples/plot_filterbank_csp_vs_csp.py
-  | tutorials/plot_Getting_Started.py
-  | tutorials/tutorial_4_adding_a_dataset.py
-  | tutorials/plot_statistical_analysis.py
-  | tutorials/tutorial_1_simple_example_motor_imagery.py
-  | tutorials/tutorial_2_using_mulitple_datasets.py
-  | tutorials/plot_explore_paradigm.py
-  | tutorials/select_electrodes_resample.py
-  | tutorials/tutorial_3_benchmarking_multiple_pipelines.py
-)
-'''
+skip-string-normalization = true  # useful for migrating codebases, but not respected in 20.8b1, see: https://github.com/psf/black/issues/1880
```

Comment:
Not sure if wanted, not useful now anyway...

Reply:
Yep, I didn't added it to initial pre-commit setup because of this issue. More than that I think double quotes is not something bad.
Alternatively (if `skip-string-normalization = true`) we will end up with mixing of quotes types in all files and all situations. That's why I think it's better to leave it default.

Reply:
Agreed.

Reply:
so I will remove that line and reformat code (once again) to double quotes